test/test_videoapi.py

```diff
@@ -178,23 +166,23 @@ def test_accurateseek_middle(self):
                 for frame in video_reader:
                     middle_num_frames += 1
 
-                self.assertTrue(middle_num_frames < num_frames)
-                self.assertAlmostEqual(middle_num_frames, num_frames // 2, delta=1)
+                assert (middle_num_frames < num_frames)
```

Comment:
Here and in various places, we don't need the surrounding parenthesis. It's a nit though, feel free not to address!